File: dart/problem001.dart
int solution() {
  var threes = new List<int>.generate(999 ~/ 3, (int index) => (index + 1) * 3);
  var fives = new List<int>.generate(999 ~/ 5, (int index) => (index + 1) * 5);
  return (threes + fives).toSet().reduce((sum, value) => sum + value);
}

void main() {
  print(solution());
}

File: js/problem001.js
function solution() {
  let sum = 0;
  for (let n = 3; n <= 999; n += 3) {
    sum += n;
  }
  for (let n = 5; n <= 995; n += 5) {
    if (n % 3 !== 0) sum += n;
  }
  return sum;
}

console.log(solution());
